src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -246,7 +246,7 @@ class SurrealPersistenceService implements PersistenceService {
     WaveScanRecord loadScanRecord(String scanId) {
         if( !scanId )
             throw new IllegalArgumentException("Missing 'scanId' argument")
-        final statement = "SELECT * FROM wave_scan:$scanId FETCH vulnerabilities"
+        final statement = "SELECT * FROM wave_scan where id == 'wave_scan:⟨$scanId⟩' FETCH vulnerabilities"
```

Comment:
what those magic brackets do?

Reply:
thats how the id stored in surrealdb `id: wave_scan:⟨sc-8c94bed81c2a999d_1⟩`